go.mod

```diff
@@ -11,8 +11,80 @@ require (
 	github.com/spf13/cobra v1.2.1
 	github.com/stretchr/testify v1.7.0
```

Comment:
Update all the above deps too. utils had a new commit recently. You can copy paste from [here](https://github.com/axelarnetwork/tm-events/pull/38/files#diff-33ef32bf6c23acb95f5902d7097b7a1d5128ca061167ec0716715b0b9eeaa5f6R6)